alchemy/src/cloudflare/certificate-pack.ts

```diff
@@ -299,7 +299,12 @@ export const CertificatePack = Resource(
           `/zones/${zoneId}/ssl/certificate_packs/${this.output.id}`,
         );
 
-        if (!deleteResponse.ok && deleteResponse.status !== 404) {
+        if (
+          !deleteResponse.ok &&
+          deleteResponse.status !== 404 &&
+          //* if certificate is mid deletion cloudflare errors with 400
+          deleteResponse.status !== 400
```

Comment:
I worry that this just isn't a great check, but IDK what else we can do here since the 400 error message is kind of bad

Reply:
Have a look at the errors and the message data in the response. They will provide an error code you can narrow this on.

Reply:
this is the response
```
{
  "success": false,
  "errors": [
    {
      "code": 1406,
      "message": "Bad response certificate service"
    }
  ],
  "messages": [],
  "result": null
}
```

I'm like 95% sure I saw this message before but I'm not sure if it was with this error code. do you know if cloudflare error codes always align with the message?

will add the check here but I'm not sure thats enough to guarantee deletion

Reply:
I don't know. Add a fat log here so we can see the response as we use it and adapt accordingly or escalate to Cloudflare.

Reply:
looking at this:
https://cfapi.centminmod.com/#certificate-packs-delete-advanced-certificate-manager-certificate-pack

it seems that the message isn't unique so I think I'm okay just checking for the specific error code

Reply:
Code should be more stable